data-base/db_main_test.go

```diff
@@ -14,36 +16,36 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var db *RecordDB
-var records = []repo.Record{
+var db *Database
+var myTime = time.Date(2023, 3, 3, 12, 0, 0, 0, time.FixedZone("", 0)) // manually set location to Greenwich
```

Comment:
1. unused variable
2. do not name variables with the prefix `my` until it's required by business logic